solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LinearModel.java

```diff
@@ -26,11 +26,11 @@
 import org.apache.solr.ltr.norm.Normalizer;
 
 /**
- * A scoring model that computes scores using a linear Support Vector Machine (SVM) algorithm.
+ * A scoring model that computes scores using a dot product.
```

Comment:
You should explicitly mention that RankSVM and Pranking are good examples of models that can be used with this.